packages/plugin-svgo/package.json

```diff
@@ -28,6 +28,6 @@
   "dependencies": {
     "cosmiconfig": "^7.0.0",
     "deepmerge": "^4.2.2",
-    "svgo": "^1.2.2"
+    "svgo": "^2.3.0"
```

Comment:
Could you update this to `^2.3.1` so that it incorporates the [css-select vulnerability fix](https://github.com/svg/svgo/pull/1485)?